uco-observable/observable.ttl

```diff
@@ -7738,10 +7734,7 @@ How does this relate to Application?
 	a owl:DatatypeProperty ;
 	rdfs:label "addressValue"@en ;
 	rdfs:comment "The value of an address."@en ;
-	rdfs:domain
```

Comment:
This looks like it has removed WiFiAddress from domain of addressValue which would be an error.

WiFiAddress and BluetoothAddress are each Facets which can have the addressValue property on them.

Reply:
The problem being addressed here is that there were two spellings of "WiFiAddress" and "WifiAddress". Since "WifiAddress" was declared as a subclass of Facet and "WiFiAddress" was not, I removed "WiFiAddress". 

When I removed "WiFiAddress" the domain references were effected, and should be replaced with "WifiAddress"

Reply:
hmm. okay. I did not notice the spelling difference. I just saw that it was removed.
I agree with the spelling fix but we obviously need to get the correct spelled version back in as a domain.
I will look into fixing that.

Reply:
Done WifiAddress added to the domain of addressValue. This will be in the turtle-cleanup-two branch.